react_native/app/screens/Settings.js

```diff
@@ -1,84 +1,121 @@
-import React, {useState, useEffect} from 'react';
-import {SafeAreaView, Text, Button} from 'react-native';
-import {useTheme} from '@react-navigation/native';
-import {createNativeStackNavigator} from "@react-navigation/native-stack";
+import React, { useState, useEffect } from "react";
+import { SafeAreaView, Text, Button, StyleSheet } from "react-native";
+import { useTheme } from "@react-navigation/native";
+import { createNativeStackNavigator } from "@react-navigation/native-stack";
 import Icon from "react-native-vector-icons/Ionicons";
 import AsyncStorage from "@react-native-async-storage/async-storage";
 import axios from "axios";
-import {AuthContext} from "../components/context";
+import { AuthContext } from "../components/context";
+import UserEdit from "./UserEdit";
 
 const Stack = createNativeStackNavigator();
 
-function SettingsMain({navigation}){
-    const {colors} = useTheme();
-    return (
-        <SafeAreaView style={{justifyContent: 'center', alignItems: 'center', flex: 1}}>
-            <Text style={{color: colors.text}}>Settings Main</Text>
-            <Button title={"Delete Account"} onPress={() => navigation.navigate('DeleteAccount')}/>
-        </SafeAreaView>
-    );
+function SettingsMain({ navigation }) {
+  const { colors } = useTheme();
+  return (
+    <SafeAreaView>
+      <UserEdit />
```

Comment:
This component is fine, but we don't need to edit the account in the settings page